packages/react-dropdown/src/index.md

```diff
@@ -107,6 +107,50 @@ const categories = [
 </Dropdown>;
 ```
 
+#### Dropdown with multi-level item list (two columns) and multiselect.
+
+With `multiselect` enabled you are able to select multiple items from the list.
+Additionnaly you may want to add `enableCategorySelection` so that you select all the items from a category.
+
+```js
+import { Dropdown } from '@quid/react-dropdown';
+import { InputText } from '@quid/react-forms';
+
+const items = [
+  { id: 10, categoryId: 'a', label: 'One' },
+  { id: 22, categoryId: 'a', label: 'Two' },
+  { id: 33, categoryId: 'b', label: 'Three' },
+  { id: 44, categoryId: 'b', label: 'Four' },
+  { id: 55, categoryId: 'b', label: 'Four', disabled: true },
+  { id: 66, categoryId: 'b', label: 'Three', disabled: true },
+  { id: 77, categoryId: 'b', label: 'Four' },
+  { id: 88, categoryId: 'b', label: 'Three' },
+  { id: 99, categoryId: 'b', label: 'Four' },
+  { id: 101, categoryId: 'b', label: 'Three' },
+  { id: 111, categoryId: 'b', label: 'Four' },
+  { id: 121, categoryId: 'b', label: 'Three' },
+  { id: 131, categoryId: 'c', label: 'One' },
+];
+
+const categories = [
+  { id: 'a', label: 'Category A' },
+  { id: 'b', label: 'Category B' },
+  { id: 'c', label: 'Category C' },
+];
+
+<Dropdown
+  items={items}
+  multiselect={true}
+  enableCategorySelection={true}
```

Comment:
I'm wondering if it wouldn't make more sense to have this boolean configurable per-category rather than globally.

We could have:

```js
	const categories = [
  { id: 'a', label: 'Category A', selectable: true },
  { id: 'b', label: 'Category B' },
  { id: 'c', label: 'Category C' },
];
```

to make only some categories selectable.

Reply:
I agree, but I still don't know if this option can actually be useful or not.

Is there any case where you want to allow multi selection, but not allow the user to quickly select a whole category?

Could we simply enable this anytime multiselect is enabled?

Reply:
I am not sure if it's useful or not. I can't think of any use-case now where we want multi-select, but not group selection. Maybe we could keep it but make it true by default?

Reply:
I removed it.